routing/routing.xml

```diff
@@ -23,29 +23,226 @@
 
 	<!-- HELP INFORMATION ABOUT FILE -->
 	<!--
-	   Description of blocks:
-	     1) <way attribute="access">
-	        Defines acceptable roads for routingProfile. Value of -1 means way impassable,
-	        value of 1 means way passable.
-	     2) <way attribute="speed" type="speed">
-	        Defines speed limits on roads in km/h (if it is not specified on the road via maxspeed tag)
-	     3) <way attribute="priority">
-	     	Priority is a preference multiplicator for the road type (consider it is an accelerator of the road)
-	     4) <point attribute="obstacle">
-	        Obstacle defines a point of the road and whether it can be passed by the vehicle.
-	        Penalty is measured in seconds, a value of -1 means the obstacle cannot be passed and a value of 0 means "no penalty"!
-	        For routing engine.
-             5) <point attribute="obstacle_time">
-                Penalty for crossing a feature on the road. It is measured in seconds.
-                Only for visual effect (ETA).
-	     		
-	   The order of specialization matters only how it was supplied :
-	      short_way, avoid_ferries, avoid_toll, avoid_motorway, avoid_unpaved
-	   
-	   Conditional values (based on parameters) can be specified as:
-	      <select value="x" t="y" v="z">
-                <if param="parameter id"/>
-              </select>
+	
+		If you want to tweak the routing engine, read this carefully.
+		
+    Deploying your own routing.xml
+    ==============================
+		
+		When you finish editing this document (or want to test on your Android device), copy this file over to:
+		sdcard/Android/data/net.osmand.plus/files/
```

Comment:
This path may differ on different versions and depending on the setting "Data storage folder"

Reply:
Also iOS version of OsmAnd exists, however it's not easy to use custom routing.xml